packages/xo-web/src/xo-app/hub/recipes/recipe.js

```diff
@@ -0,0 +1,113 @@
+import * as ComplexMatcher from 'complex-matcher'
+import _ from 'intl'
+import ActionButton from 'action-button'
+import ButtonLink from 'button-link'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import marked from 'marked'
+import React from 'react'
+import { Card, CardBlock, CardHeader } from 'card'
+import { escapeRegExp } from 'lodash'
+import { form } from 'modal'
+import { connectStore } from 'utils'
+import { createGetObjectsOfType } from 'selectors'
+import { createKubernetesCluster } from 'xo'
+import { injectState, provideState } from 'reaclette'
+import { success } from 'notification'
+import { withRouter } from 'react-router'
+
+import RecipeForm from './recipe-form'
+
+const RECIPE_INFO = {
+  id: '05abc8a8-ebf4-41a6-b1ed-efcb2dbf893d',
+  name: 'Kubernetes cluster',
+  description:
+    'Creates a Kubernetes cluster composed of 1 master and a configurable number of nodes working for the master.',
+}
+
+export default decorate([
+  withRouter,
+  connectStore(() => ({
+    pools: createGetObjectsOfType('pool'),
+    recipeCreatingResources: state => state.recipeCreatingResources,
+  })),
+  provideState({
+    initialState: () => ({
+      selectedInstallPools: [],
+    }),
+    effects: {
+      async create() {
+        const { markRecipeAsCreating, markRecipeAsCreated } = this.props
+        const recipeParams = await form({
+          defaultValue: {
+            pool: {},
+          },
+          render: props => <RecipeForm {...props} />,
```

Comment:
You can use the `component` prop instead.

Reply:
`component` prop was never used in `XO`, I tried to use it in different ways but it doesn't work. Are you sure about this information ?

Reply:
What's not working? I guess it should work otherwise I don't know why it would even exist. Maybe the props aren't correctly passed to the component?

Reply:
Bug repored in https://github.com/vatesfr/xen-orchestra/issues/4705